src/client/cli/client.cpp

```diff
@@ -52,8 +53,9 @@ namespace mp = multipass;
 namespace mpl = multipass::logging;
 
 mp::Client::Client(ClientConfig& config)
-    : cert_provider{std::move(config.cert_provider)},
-      rpc_channel{mp::client::make_channel(config.server_address, config.conn_type, *cert_provider)},
+    : rpc_channel{config.conn_type == mp::RpcConnectionType::ssl
+                      ? mp::client::make_secure_channel(config.server_address, config.cert_provider.get())
+                      : mp::client::make_insecure_channel(config.server_address)},
```

Comment:
This sounds scary, why would we ever have a non-SSL connection?

Reply:
For testing.  It's always been there, it's just I broke it out to deal with things better.

Reply:
I see now, yeah.